cockatrice/src/userinfobox.cpp

```diff
@@ -11,20 +11,19 @@
 
 #include <QDateTime>
 #include <QGridLayout>
-#include <QHBoxLayout>
 #include <QLabel>
 #include <QMessageBox>
 
-const qint64 SIXTY = 60;
-const qint64 HOURS_IN_A_DAY = 24;
-const qint64 DAYS_IN_A_YEAR = 365;
+const quint64 SIXTY = 60;
+const quint64 HOURS_IN_A_DAY = 24;
+const quint64 DAYS_IN_A_YEAR = 365;
```

Comment:
>wait, why do we do something like assigning SIXTY as a constant, like what

I assume the intention was to name it `MINUTES_IN_AN_HOUR` when looking at the other constant names.